apps/heft/src/templates/heft.json

```diff
@@ -44,61 +47,79 @@
     //
     // {
     //   /**
-    //    * The kind of built-in operation that should be performed.
+    //    * (Required) The kind of built-in operation that should be performed.
     //    * The "copyFiles" action copies files that match the specified patterns.
     //    */
     //   "actionKind": "copyFiles",
     //
     //   /**
-    //    * The stage of the Heft run during which this action should occur. Note that actions specified in heft.json
-    //    * occur at the end of the stage of the Heft run.
+    //    * (Required) The Heft stage when this action should be performed.  Note that heft.json event actions
+    //    * are scheduled after any plugin tasks have processed the event.  For example, a "compile" event action
+    //    * will be performed after the TypeScript compiler has been invoked.
+    //    *
+    //    * Options: "pre-compile", "compile", "bundle", "post-build"
```

Comment:
"clean" is also an option, though we don't support it for the copy plugin. If this is just meant to show how that specific plugin is used, then ignore this

Reply:
It is. I did include "clean" as an option for the `deleteGlobs` action type.